lib/routes/charge/pos_payment_dialog.dart

```diff
@@ -61,41 +80,56 @@ class _PosPaymentDialogState extends State<PosPaymentDialog> {
     super.dispose();
   }
 
-  Widget _actionsWidget(){
-    return Row(
-      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-      children: <Widget>[
-        _clearSaleButton(), _cancelButton()
-      ],
-    );
-  }
-
-  Widget _cancelButton() {
-    return FlatButton(
-        padding: EdgeInsets.only(top: 8.0, bottom: 16.0, left: 0.0),
-        child: Text(
-          'CANCEL',
-          textAlign: TextAlign.center,
-          style: Theme.of(context).primaryTextTheme.button,
-        ),
-        onPressed: () {
-          Navigator.of(context).pop(PosPaymentResult());
-        },
-      );
-  }
-
-  Widget _clearSaleButton() {
-    return FlatButton(
-        padding: EdgeInsets.only(top: 8.0, bottom: 16.0, right: 0.0),
-        child: Text(
-          'CLEAR SALE',
-          textAlign: TextAlign.center,
-          style: Theme.of(context).primaryTextTheme.button,
-        ),
-        onPressed: () {
-          Navigator.of(context).pop(PosPaymentResult(clearSale: true));
-        },
-      );
+  @override
+  Widget build(BuildContext context) {
+    return AlertDialog(
+        titlePadding: EdgeInsets.fromLTRB(20.0, 22.0, 0.0, 8.0),
+        title: _syncedToChain
```

Comment:
Why use a class member for this that is assigned bellow in another component?
It is a lot cleaner if the account stream builder should be here on top and pass this value to the `buildWaitingPayment` function bellow.

Reply:
[Done](https://github.com/breez/breezmobile/pull/311/commits/82828126ea7fdab59a30b9c12e6138c79e4f9893)